### ISSUE TITLE
Failed to load resource: the server responded with a status of 404 ()

### ISSUE THREAD

**Issue by fullastern** [DanielsWrath/YNABGoingDutch#3]
Utility is not working at the moment.
This is the console output in Chrome:

Failed to load resource: the server responded with a status of 404 ()

Also tried with Firefox, Edge and IE. No success.

**Comment by fullastern** [comment]
Hi,

I don’t know what the 404 is referring to and I can’t check right now.
The CSV is from ING.

**Comment by RubyTunes** [comment]
Hey Daniel, excited to try this but I also am experiencing the same issue as fullastern. Console shows 404 for dropzone.css as soon as I load the page. Upon uploading a file, I can't click the checkmark.
<img width="738" alt="screen shot 2018-07-08 at 2 47 07 pm" src="https://user-images.githubusercontent.com/40951804/42419931-246e5524-82be-11e8-9481-024ede71fe10.png">

**Comment by DanielsWrath** [comment]
Yes, this seems to be an issue with Windows. On macOS it is working fine. I have to develop on a Windows machine to fix it. I will look into it asap.

It shouldnt show the checks at all, as it should process it locally and then delete it from the queue.

**Comment by RubyTunes** [comment]
Hey Daniel, actually I'm on macOS also, so that doesn't seem to be the issue.

**Comment by DanielsWrath** [comment]
Oh ok, weird that it works for me then. Thanks for letting me know!

**Comment by DanielsWrath** [comment]
@RubyTunes @fullastern Please check out the updated website. Could you check if it works now?

**Comment by RubyTunes** [comment]
@DanielsWrath this works perfectly for me now! Thank you so much, this is going to save a lot of time!
